### ISSUE TITLE
[grid] One wide column not inline when on tiny screen

### ISSUE THREAD

**Issue by daveharris** [fomantic/Fomantic-UI#2856]
# Bug Report
With a screen size of 447px and smaller, when a grid contains a `one wide column`, the subsequent `column` is forced onto the next line, even though the number of columns is exactly 16. Displaying on a screen size larger than 448px displays correctly. Changing to `two wide column` (and `fourteen wide column`) also works correctly. 

## Steps to reproduce
1. Create a `ui grid` containing a `one wide column` and a `fifteen wide column`
2. Reduce your window width to 447px

## Expected result
The `fifteen wide column` is displayed **next to** the `one wide column`

## Actual result
The `fifteen wide column` is displayed **under** the `one wide column`. You can see on the jsfiddle that the second `ui grid` with a `two wide column` and a `fourteen wide column` always displays correctly.

## Testcase
https://jsfiddle.net/szxkdre1/1/

## Screenshot (if possible)
![One wide column](https://github.com/fomantic/Fomantic-UI/assets/111275/ba0c896e-0a48-4f92-bbce-7379d9f41ab8)

## Version
2.9.2

**Comment by JaiminR28** [comment]
can i work upon it ?

**Comment by lubber-de** [comment]
Sure

**Comment by JaiminR28** [comment]
I think this issue has been solved in the version 2.9.3 .

**Comment by daveharris** [comment]
@JaiminR28 I don't think so, [the JSFiddle](https://jsfiddle.net/xvzns35r/1/) is referencing https://fomantic-ui.com/dist/semantic.js and https://fomantic-ui.com/dist/semantic.css which is currently at `2.9.3`:

```css
/*
 * # Fomantic UI - 2.9.3
 * https://github.com/fomantic/Fomantic-UI
 * https://fomantic-ui.com/
 ...
```

**Comment by JaiminR28** [comment]
ok i'll try to look into it.
